conf/uppmax-modules.config

```diff
@@ -0,0 +1,37 @@
+/*
+vim: syntax=groovy
+-*- mode: groovy;-*-
+ * -------------------------------------------------
+ *  Nextflow config file with environment modules for UPPMAX
+ * -------------------------------------------------
+ */
+
+singularity {
+  enabled = false
+}
+
+process {
```

Comment:
This still has the old process$ syntax you replaced further up in the other configs - maybe that would be good here too?